source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -230,7 +231,7 @@
             if (calification.Califications) {
                 for (var i in calification.Califications) {
                     var cal = calification.Califications[i];
-                    valuesByKey[cal.Key] = parseFloat(cal.Value.toFixed(1));
+                    valuesByKey[cal.Key] = cal.Value && parseFloat(cal.Value.toFixed(1));
```

Comment:
Another option is to avoid to add a _valuesByKey_ when it has not value, I am not sure about pros and cons.